tests/utils.go

```diff
@@ -4416,7 +4416,7 @@ func resetToDefaultConfig() {
 		// we can just skip the restore step.
 		return
 	}
-
+	RestoreKubeVirtResource()
```

Comment:
`resetToDefaultConfig` is reset after/before each each test. `RestoreKubeVirtResource` is supposed to be only called once after the test suite is done, and that happens in `SynchronizedAfterTestSuiteCleanup`.

The issue we have is that `UpdateKubeVirtConfigValueAndWait` only resets `spec.configuration` to the state from before the test run, while it should also reset other settings, like for certificates.

Reply:
@rmohr Thank you for pointing that out. I've adopted the code